### ISSUE TITLE
IPv6 Support

### ISSUE THREAD

**Issue by ssamjh** [Nyr/openvpn-install#183]
Do you have any plans to support IPv6 with this script?

**Comment by Nyr** [comment]
This is not really possible or viable at this point with most VPS providers.

Previous issues had been opened about this if you want further detail.

**Comment by ssamjh** [comment]
So from what I can guess, most of the tickets that have been opened, people have been wanting to connect to their server with IPV6, but what I am looking for is allowing IPV6 connectivity when connected to the VPN.

**Comment by Nyr** [comment]
> what I am looking for is allowing IPV6 connectivity when connected to the VPN.

That's the issue actually. Connecting via IPv6 is simple. Providing IPv6 to clients needs either a fully routable subnet (which isn't available for most users with IPv6) or IPv6 NAT (which is an ugly workaround and only recently supported).

TL;DR: no IPv6 in the near future.
